yml/apdb.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: "ApdbSchema"
 "@id": "#apdbSchema"
-version: "0.1.1"
+version: "1.0.0"
```

Comment:
Should this be 1.0 and not, say, 0.2? What makes this version more or less "pre-release" than the old one?

Reply:
We're using a sort-of semantic versioning system here, and this is a fully breaking non-backwards compatible change, hence the major number bump. Version 0.1.1 is the first "there are versions", while anything before that is considered 0.1.0.

https://dmtn-269.lsst.io#version-compatibility

Reply:
Semantic versioning does not require that versions 0.x be mutually compatible, so that's no argument either way.

Reply:
See the system we're using (following the one in daf_butler) in the link above: this i a major number bump, per that definition (@andy-slac can correct me if I'm wrong here).

Reply:
I need to add that the exact meaning of the minor version number is not so well-defined now. Major and patch roles are very clear as non-compatible and compatible (and change on this PR is clearly non-compatible). Minor version role is defined now as "backward-compatible", but it is not super-clear for now what that means, as it probably implies that something on the client side may be responsible for that compatibility. In Butlerland we define minor version compatibility as "read-compatible", but for APDB that notion probably does not matter at all as APDB's primary client is AP pipelines which are always writing.

Anyways, I think we may eventually figure out minor version meaning, for now I expect changes on schema side that are either non-compatible or fully-compatible.

Additionally, PPDB schema is supposed to reflect APDB schema, so there are probably similar versioning issues on that side, but I'm not ready to discuss those now. And potential schema migration for PPDB will be a separate huge topic.

Reply:
For what it's worth, for the alerts schemas we are using the FORWARD_TRANSITIVE type of the [Confluent compatibility model](https://docs.confluent.io/current/schema-registry/docs/avro.html#forward-compatibility)--see https://dmtn-093.lsst.io/